### PR TITLE
MariaDB on-prem_Fixed UAT Bugs

### PR DESCRIPTION
**Pull request by BalramVerma1**
Signed-off-by: balram verma <balram.verma@ibm.com>

### CODE REVIEW

File: filter-plugin/logstash-filter-mariadb-guardium/README.md [hunk 1]
```diff
@@ -153,10 +152,11 @@ at Filebeat level only.
 Filebeat provides a solution which can be implemented to ignore such logs, below are the steps
 1. Open filebeat.yml in notepad or any OS specific text editor.
 2. Search "filebeat.inputs" property.
-3. Under the "filebeat.inputs" property, uncomment the exculde_lines property and place the query user wants to ignore at filebeat level
+3. Under the "filebeat.inputs" property, uncomment the exclude_lines property and place the query user wants to ignore at filebeat level
+4. Different third party tool may generate different system queries, so based on observation user need to modify queries in 'exclude line' of filebeat configuration.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mariadb-guardium/README.md:156]:
```suggestion
4. Different third party tools may generate different system queries, so modify queries in the 'exclude line' of your Filebeat configuration as needed.
```

File: filter-plugin/logstash-filter-mariadb-guardium/README.md [hunk 2]
```diff
@@ -165,13 +165,13 @@ The Guardium universal connector is the Guardium entry point for native audit lo
...
 * You must have permission for the S-Tap Management role. The admin user includes this role by default.
-* Download the [logstash-filter_offline_plugin_mariadb.zip](MariaDBOverFilebeatPackage/logstash-filter-mariadb_guardium_filter.zip) plug-in.
+* Download the [logstash-filter-mariadb_guardium_filter.zip](MariaDBOverFilebeatPackage/logstash-filter-mariadb_guardium_filter.zip) plug-in.
 * Download the plugin filter configuration file [MariaDB.conf](MariaDB.conf).
 
 ## Procedure
 1. On the collector, go to Setup > Tools and Views > Configure Universal Connector.
 2. Enable the connector if it is disabled before uploading the UC plug-in.	
-3. Click Upload File and select the offline [logstash-filter_offline_plugin_mariadb.zip](MariaDBOverFilebeatPackage/logstash-filter-mariadb_guardium_filter.zip) plug-in. After it is uploaded, click OK.
+3. Click Upload File and select the offline [logstash-filter-mariadb_guardium_filter.zip](MariaDBOverFilebeatPackage/logstash-filter-mariadb_guardium_filter.zip) plug-in. After it is uploaded, click OK.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mariadb-guardium/README.md:174]:
```suggestion
3. Click ```Upload File``` and select the offline [logstash-filter-mariadb_guardium_filter.zip](MariaDBOverFilebeatPackage/logstash-filter-mariadb_guardium_filter.zip) plug-in. After it is uploaded, click ```OK```.
```

File: filter-plugin/logstash-filter-mariadb-guardium/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 # Changelog
 Notable changes will be documented in this file.
 
+## [2.3.70]
+- Downgraded gradle version to 6.5.1
+
```
Comment (Tal-Daniel) [inline filter-plugin/logstash-filter-mariadb-guardium/CHANGELOG.md:6]: @BalramVerma1, could you state the main changes in this version? before the downgrade, I'd add for example, the \n\r change, and also the shift from excluding in connector configuration to excluding at FileBeat level.

Explain in simple and compact English sentence please, for non-developers, if possible.

File: filter-plugin/logstash-filter-mariadb-guardium/src/main/java/com/ibm/guardium/mariadb/ParserHelper.java [hunk 4]
```diff
@@ -257,7 +257,7 @@ public static String parseSQL(final Event event) throws Exception {
 				&& event.getField(ApplicationConstant.OBJECT_KEY).toString().trim() != "") {
 			builder = new StringBuilder(event.getField(ApplicationConstant.OBJECT_KEY).toString().trim());
 			builder.deleteCharAt(builder.length() - 1).deleteCharAt(0);
-			fullSql = builder.toString().replaceAll("\n|\r", " ").replaceAll("\\\\n|\\\\r", " ").replaceAll("\\\\'", "'").replaceAll("\\\\","").replaceAll("\\\\t", "");
+			fullSql = builder.toString().replaceAll("\\\\n","\n").replaceAll("\\\\r","\r").replaceAll("\\\\'", "'").replaceAll("\\\\","").replaceAll("\\\\t","\t");
```
Comment (Tal-Daniel) [inline filter-plugin/logstash-filter-mariadb-guardium/src/main/java/com/ibm/guardium/mariadb/ParserHelper.java:260]: explain in CHANGELOG (see comment above),  and in commit message.

**Review by Tal-Daniel — CHANGES_REQUESTED** [review]

File: filter-plugin/logstash-filter-mariadb-guardium/CHANGELOG.md (hunk 3 above)
Comment (BalramVerma1) [reply filter-plugin/logstash-filter-mariadb-guardium/CHANGELOG.md]: done

**Review by Tal-Daniel — APPROVED** [review]

**Review by RefaelAdi — APPROVED** [review]

**Review by ofer-haim — APPROVED** [review]

**Pull request merged**